InternetHospitalWebUI/src/app/app.component.html

```diff
@@ -1,12 +1,14 @@
 <!--The content below is only a placeholder and can be replaced.-->
-<div class="container">
+<!-- <div class="container">
   <div class="row">
       <app-header></app-header>
   </div>
   <div class="row">
-    <router-outlet></router-outlet>
```

Comment:
why you delete  <router-outlet></router-outlet>?

Reply:
This was made by mistake.
Brought it back in newer commit